### PR TITLE
[fix] indentation for hypothesis profile and proper inheritance for QuantizationTestCase

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#25934 [fix] indentation for hypothesis profile and proper inheritance for QuantizationTestCase**

Summary:
att

Test Plan:
python test test/test_quantized_nn_mods.py

Reviewers:
pt1quant
Subscribers:

Tasks:

Tags:

Differential Revision: [D17318270](https://our.internmc.facebook.com/intern/diff/D17318270)

### CODE REVIEW

File: test/common_quantization.py [hunk 1]
```diff
@@ -60,7 +60,11 @@ def prepare_dynamic(model, qconfig_dict=None):
 
 # QuantizationTestCase used as a base class for testing quantization on modules
 class QuantizationTestCase(TestCase):
+    def __init__(self, method_name='runTest'):
```
Comment (bddppq) [inline test/common_quantization.py]: This shouldn't be needed

**Review by bddppq — APPROVED** [review]
LGTM

**Comment by ezyang** [comment]
How was this tested?

**Comment by jerryzh168** [comment]
@ezyang tested locally by enabling pytorch_ci profile and printing derandomize variable in the test case

**Pull request closed without merging**